### PR TITLE
update download page

### PR DESCRIPTION
**Pull request by MartinNowak**
- remove dropped FreeBSD-32 builds
- fix FreeBSD info links
- remove unmaintained nightly release channel

Also see https://dlang.org/changelog/2.094.0.html#ldc-built-releases.

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

File: download.dd [hunk 1]
```diff
@@ -114,49 +114,17 @@ $(DIVC download_channels,
         $(SBTN $(B_RPM32), i386) $(SBTN $(B_RPM64), x86_64) $(SBTN $(B_ARCH linux, tar.xz), tar.xz)
     )
 
-    $(DOWNLOAD openSUSE, $(OPENSUSE), linux, Linux,
+    $(DOWNLOAD openSUSE, $(OPENSUSE), freebsd, FreeBSD,
```
Comment (wilzbach) [inline download.dd:117]: Shouldn't this stay `linux, Linux`?

Comment (MartinNowak) [reply download.dd:117]: My bad, fixed.

**Comment by CyberShadow** [comment]
> remove dropped FreeBSD-32 builds

The links seem to still be working. Should this be merged when 2.094 is out OSLT?

**Comment by MartinNowak** [comment]
> The links seem to still be working. Should this be merged when 2.094 is out OSLT?

It's already wrong for the beta, old releases can be found in the archive, doesn't seem a good reason to wait with this.

**Pull request merged**